schema/core.md

```diff
@@ -595,10 +595,12 @@ Multiple files can be expressed as:
 }
 ```
 
-### Data Collections
+### Has part and is part of
 
 Collections of records can be expressed using the **hasPart** (and inverse
-**isPartOf**) properties of `CreativeWork`.
+**isPartOf**) properties of `CreativeWork`. These properties are used to show
+a relationship between two or more records, along with `identifier` property to 
```

Comment:
I think this needs to be more specific. "hasPart" and "isPartOf" implies a specific type of relationship - e.g., that of a collection (hasPart) and a collected item (isPartOf). This construct should not be used to express different relationships - e.g., this record references another record, which should be handled via schema:citation.

I suggest: "These properties are used to show specific relationships between a collection record and its member records where **hasPart** is used on the collection to indicate that it contains other records, and **isPartOf** is used on a collected record to indicate that it is part of a collection. The `identifier` property is used to uniquely identify each record."

Reply:
Thanks! I added your suggestion.